tests/core_tests/chaingen.h

```diff
@@ -544,7 +551,11 @@ struct push_core_event_visitor: public boost::static_visitor<bool>
 
     cryptonote::tx_verification_context tvc = AUTO_VAL_INIT(tvc);
     size_t pool_size = m_c.get_pool_transactions_count();
+<<<<<<< HEAD
     m_c.handle_incoming_tx({t_serializable_object_to_blob(tx), crypto::null_hash}, tvc, m_txs_keeped_by_block, false, false);
+=======
+    m_c.handle_incoming_tx(t_serializable_object_to_blob(tx), tvc, m_txs_keeped_by_block, false, m_txs_do_not_relay);
+>>>>>>> Fix mempool key image privacy bug
```

Comment:
Leftover conflict marker

Reply:
What diff is this from? It does not appear in the most up-to-date version.